test/unit/controllers/payment_reference/post_product_reference_controller_test.js

```diff
@@ -105,7 +105,7 @@ describe('product reference post controller', function () {
       nock(config.PRODUCTS_URL).get(`/v1/api/products/${product.external_id}`).reply(200, product)
       nock(config.ADMINUSERS_URL).get(`/v1/api/services?gatewayAccountId=${product.gateway_account_id}`).reply(200, service)
 
-      const referenceNumber = 'This_is_a_256_characters_long_String_This_is_a_256_characters_long_String_This_is_a_256_characters_long_String_This_is_a_256_characters_long_String_This_is_a_256_characters_long_String_This_is_a_256_characters_long_String_This_is_a_256_characters_long_Stri'
+      const referenceNumber = 'This_is_a_51_characters_long_String_This_is_a_51_ch'
```

Comment:
Missed a trick by not using `'This string is exactly 51 characters long in length'`!